dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -821,4 +850,28 @@ def test_histogram_option_integration(self):
         options.histogram_and_quantiles.method = ["sturges", "doane"]
         num_profiler = FloatColumn(name="test2", options=options)
         self.assertIsNone(num_profiler.histogram_selection)
-        self.assertEqual(["sturges", "doane"], num_profiler.histogram_bin_method_names)
+        self.assertEqual(["sturges", "doane"],
+                         num_profiler.histogram_bin_method_names)
+
+        # test hisogram bin count set
+        options.histogram_and_quantiles.method = None
+        options.histogram_and_quantiles.hist_bin_count = 100
+        num_profiler = FloatColumn(name="test3", options=options)
+        self.assertIsNone(num_profiler.histogram_selection)
+        self.assertEqual(['custom'], num_profiler.histogram_bin_method_names)
```

Comment:
Should we make `.histogram_selection = 'custom'` in this case? I'm just thinking for in terms of accessing that dictionary that is indexed via names. I suppose this will eventually get picked by picking the best method out of a list of 1, but currently my implementation for user picking 1 method is it sets `.histogram_selection` in `__init__`. I'm all for changing that to be None to match the custom case, I just think they should be consistent.

Reply:
Hmm, thinking about this. I wonder if it should be `None`, bc technically nothing has been computed yet and would also fit what the list is like.

I'm open to change, but rn i'm coding such that default is none till update is run

Reply:
Well then do you think we should make the case of the user picking one method match this? Or do you think that isn't as important.